File: section_StepDefinitions.js
const { Given, When, Then }= require('@cucumber/cucumber')

// Scenario: Simple User successfully login
Given('Simple Go to login new page', function () {
  // Your test goes here ;
});

When('Simple Enter user email {string} and password {string}', function (str1, str2) {
  //Simple Enter user email "aman@gmail" and password "aman@1234"
  // Your test goes here ;
});

Then('Simple Click on login button', function () {
// Your test goes here ;
});
Then('Simple Login successfully {string}', function (str1) {
//Simple Login successfully "uimatic@gmail.com"
// Your test goes here ;
});

File: section_StepDefinitions.cs
using TechTalk.SpecFlow
namespace TempNameSpace.Steps
{
    [Binding]
    public class UserRegistrationSteps
    {
        [Given(@"I am on the registration page")]
        public void GivenIAmOnTheRegistrationPage() {
            // Your test goes here ;
        }
        [When(@"I fill in the registration form with valid details")]
        public void WhenIFillInTheRegistrationFormWithValidDetails(Table table) {
            // Your test goes here ;
        }
        [And(@"I submit the registration form")]
        public void AndISubmitTheRegistrationForm() {
            // Your test goes here ;
        }
        [Then(@"I shoulad see a success message")]
        public void ThenIShouladSeeASuccessMessage() {
            // Your test goes here ;
        }
        [And(@"I should be logged in")]
        public void AndIShouldBeLoggedIn() {
            // Your test goes here ;
        }
    }
}